scripts/portstat

```diff
@@ -2,18 +2,22 @@
 
 #####################################################################
 #
-# portstat is a tool for summarizing network statistics. 
+# portstat is a tool for summarizing network statistics.
 #
 #####################################################################
 
 import argparse
 import cPickle as pickle
 import datetime
+import getopt
 import os.path
+import re
+import subprocess
 import swsssdk
 import sys
 import time
 
+from pprint import pprint
```

Comment:
Please move this line to be in alphabetical order (i.e., after `from natsort import natsorted`)

Reply:
removed pprint.  I was using for development/debugging and forgot to remove.